Ngoctbt/fix

tranbuituanngoc opened the pull request.

tranbuituanngoc committed:
feat(hotel): add crud+search hotel

- add crud, search hotel
- add paging hotel
- add role permission for sidebar
- add update last_login_at for user
- add model, service, repository for user, role, hotel, city

tranbuituanngoc committed:
feat(user): add CRUD user function

tranbuituanngoc committed:
feat(profile): add profile function

tranbuituanngoc committed:
feat(role): add CRUD role function

tranbuituanngoc committed:
fix(city): fix hotel, city

- add city Service
- fix hotel controller

tranbuituanngoc committed:
fix(image): add folder images

tranbuituanngoc committed:
fix(project): fix notification

tranbuituanngoc committed:
feat(docker): add docker config

buutv commented on app/Http/Controllers/HotelController.php:
```diff
@@ -3,43 +3,56 @@
     protected $hotelService;
 
-    public function __construct(CityRepository $cityRepository, HotelService $hotelService)
+    const ADMIN_ROLE = 'ADMIN';
```
@tranbuituanngoc constant thì em tạo file rồi dùng chung nhé, hôm trước anh có hướng dẫn rồi.

buutv commented on app/Http/Controllers/ProfileController.php:
```diff
@@ -37,22 +39,26 @@ public function index()
+            $this->userService->updateProfile($data, $user->id);
 
-        return redirect()->route('profile.index');
+            return redirect()->route('profile.index')->with('success', 'Profile updated successfully.');;
```
@tranbuituanngoc msg tạo file resource rồi dùng, hôm trước anh cũng hướng dẫn rồi.

buutv reviewed (commented):
@tranbuituanngoc 
1. Em tạo file constant rồi define và dùng chung nhé.
2. Message thì tạo file trong resource rồi dùng như anh hướng dẫn hôm trươc.

tranbuituanngoc committed:
Merge pull request #7 from tranbuituanngoc/ngoctbt/docker

feat(docker): add docker config

tranbuituanngoc committed:
fix(message, middleware, hotel): fix

- fix create hotel
- fix middleware for checkRole
- create constant file
- create message resource file

buutv commented:
@tranbuituanngoc nhiều files quá tạm thời anh merge để em test nha, bữa sau join dự án thì em tách ra nhiều pull

buutv commented:
@tranbuituanngoc anh k0 có quyền merge nên em merge luôn nhé
